src/NECompletion-Tests/NECControllerTest.class.st

```diff
@@ -5,27 +5,205 @@ Class {
 	#name : #NECControllerTest,
 	#superclass : #TestCase,
 	#instVars : [
-		'controller'
+		'controller',
+		'editor'
 	],
 	#category : #'NECompletion-Tests'
 }
 
+{ #category : #'tests-keyboard' }
+NECControllerTest >> allSelfClosingCharacters [
+	^ {'{}' . '[]' . '()' . '<>' . '||' . '""' . ''''''}
+]
+
+{ #category : #'tests-keyboard' }
+NECControllerTest >> allSmartCharacters [
+	^ self allSelfClosingCharacters , { '||'. '<>' }
+]
```

Comment:
Isn't this a duplication? I mean, the '||' and '<>' are twice there...

Reply:
To be clear - this change should be transparent to everyone. You will only notice a difference if you highlight some text and press | or < (and any of the existing closing characters ],},)

Reply:
What do i need to do to get this PR resolved?

Reply:
I restarted the tests check. It seems to work fine.